api/integrations/git/post-receive

```diff
@@ -51,9 +52,11 @@ client = zulip.Client(
 
 # check_output is backported from subprocess.py in Python 2.7
 def check_output(*popenargs, **kwargs):
+    # type: (*Any, **Any) -> Any
     if 'stdout' in kwargs:
         raise ValueError('stdout argument not allowed, it will be overridden.')
-    process = subprocess.Popen(stdout=subprocess.PIPE, *popenargs, **kwargs)
+    process = subprocess.Popen(stdout=subprocess.PIPE, *popenargs, **kwargs) # type: ignore
+    # ^ Maybe too many arguments (3 instead of 2)? https://chat.zulip.org/#narrow/near/72687/stream/GCI.20help/topic/mypy
```

Comment:
This may be a bug with the subprocess.Popen stubs; @rwbarton does that seem plausible?  I think what's going on is that we're doing something wrong here, namely passing a kwarg before *popenargs...

Anyway I think for Zulip, we no longer support Python 2.6, so quite possibly the write answer is to just remove this hack instead of annotating it...